src/main/resources/log4j.properties

```diff
@@ -11,22 +10,16 @@ log4j.appender.RollingAppender.DatePattern='.'yyyy-MM-dd
 log4j.appender.RollingAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.RollingAppender.layout.ConversionPattern=[%p] %d %c %M - %m%n
 
-log4j.appender.RollingAppenderU=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.RollingAppenderU.File=log/spark-verbose.log
-log4j.appender.RollingAppenderU.DatePattern='.'yyyy-MM-dd
-log4j.appender.RollingAppenderU.layout=org.apache.log4j.PatternLayout
-log4j.appender.RollingAppenderU.layout.ConversionPattern=[%p] %d %c %M - %m%n
-
+# Ingestion 3 logger
+# A FileAppender should be dynamically added to this logger
+log4j.logger.ingestion3=DEBUG, stdout
 
-# By default, everything goes to console and file
-log4j.rootLogger=ERROR, RollingAppender, stdout
+# By default, everything goes to spark.log file
+log4j.rootLogger=DEBUG, RollingAppender
```

Comment:
Do we want it to log at DEBUG ordinarily? That might actually slow things down.

Reply:
Honestly, I really wasn't sure but I can roll it back ...